### DISCUSSION TITLE
使用DBeaver导出Oracle数据，导出的写法不支持Oracle多个values的写法，导出的是mysql支持的insert语法

### DISCUSSION THREAD

**Discussion by ttr5966** [dbeaver/dbeaver#19619] [Q&A] [answered]
### Description

使用DBeaver导出Oracle数据，导出的写法不支持Oracle多个values的写法，导出的是mysql支持的insert语法
![image](https://user-images.githubusercontent.com/61843308/227898649-dca3fff7-71d9-46c6-b5b1-eee2e0158b9b.png)

### DBeaver Version

22.1.4

**Comment by LonwoLonwo** [comment]
Hello @ttr5966 

Please translate your ticket into English.

**Comment by E1izabeth** [comment]
Could you describe detailed steps to reproduce in English? How exactly are you trying to do export?

Do you use this export to SQL?
![image](https://user-images.githubusercontent.com/28875055/228785956-21f96341-5fa2-4655-b6fb-0e293ed626c9.png)

Any footage would be useful.

**Comment by ttr5966** [comment]
Use DBeaver export Oracle data, export of writing does not support multiple Oracle values of wording, insert syntax is mysql support is derived
![image](https://user-images.githubusercontent.com/61843308/229728903-b7b39476-63c2-47e1-aecb-a012b6b90051.png)
This is what I chose

**Comment by ttr5966** [comment]
Mine is Oracle Database，The normal export should be a single insert statement, because Oracle does not support multiple values imports
![image](https://user-images.githubusercontent.com/61843308/229730300-7df8b369-32f1-4728-8081-341b8b2e3a43.png)
This is the data I just exported today, and the format of values is the same

**Comment by ttr5966** [comment]
导出应该是这样结构的
![image](https://user-images.githubusercontent.com/61843308/229758631-541d448c-8d9d-4173-8746-bb64442c941f.png)

**Comment by uslss** [comment, marked as answer]
@ttr5966 the number of rows per statement could be changed in preferences
![image](https://user-images.githubusercontent.com/31996417/230023037-ebca6db1-7a4b-4286-bc86-ce02da457a6b.png)

**Comment by ttr5966** [comment]
> @ttr5966 the number of rows per statement could be changed in preferences ![image](https://user-images.githubusercontent.com/31996417/230023037-ebca6db1-7a4b-4286-bc86-ce02da457a6b.png)
Thank you！！！
